applications/FSIapplication/python_scripts/fsi_analysis.py

```diff
@@ -77,17 +77,52 @@ def Initialize(self):
         self.ModifyInitialProperties()
         self.ModifyInitialGeometry()
 
-        self._GetListOfProcesses()
-        self._SetUpAnalysis()
+        # Initialize the user-provided processes
+        self._AnalysisStage__CreateListOfProcesses() # Why name mangling?
```

Comment:
hm maybe we should make this protected such that you can access it

Reply:
This was my thought as well. I did this as a temporary solution to avoid modifying the Analysis Stage in this PR.